app/models/scenario/user_updates.rb

```diff
@@ -98,6 +98,11 @@ def validate_no_yaml_error
     end
   end
 
+  # Validation method for when a user sets their metadata.
+  def validate_metadata_size
+    errors.add(:metadata, 'can not exceed 64Kb') if metadata.to_s.bytesize > 64_000
```

Comment:
Was 64_000 chosen intentionally, rather than 65_536? I doubt this would be an issue in practice, but it might confuse someone that we say the limit is 64Kb but return an error when storing slightly less.

Reply:
Good catch!